reporter/sources.py

```diff
@@ -355,6 +355,8 @@ class DBQueryErrorsSource(PHPLogsSource):
 * {backtrace}
 """
 
+    ER_LOCK_DEADLOCK = 1213
```

Comment:
from the literal "1213" which in Python is converted to an integer in case it doesn't overflow the int32/int64 ;-)